### PR TITLE
Tutorial.css max width #1229

### PR DESCRIPTION
**Pull request by edyedy123**
Hey,
In tutorial.css I added a img property and set the max width to 100% of the container size, this should stop images from being wider than the tutorial container.
img{
  max-width: 100%;
}

### CODE REVIEW

**Comment by gideonthomas** [comment]
Hi @edyedy123, can you reference the issue number that this fixes?

**Comment by flukeout** [comment]
Will check this out later today, thanks.

File: public/resources/tutorial/tutorial.css [hunk 1]
```diff
@@ -135,6 +135,10 @@ li {
   margin: 0 0 10px 0;
 }
 
+img{
```
Comment (flukeout) [inline public/resources/tutorial/tutorial.css]: Please add a space between the tag name and the open bracket like this...

``img {``

**Comment by flukeout** [comment]
Hey @edyedy123 - I tested this locally and it works, fine. I left a small comment about adding a space between the tag name and the opening bracket. Once you get that in we'll merge it into the codebase. Thank you!

**Comment by gideonthomas** [comment]
Thanks for the contribution @edyedy123! I'll merge this in now.

**Pull request merged**